### ISSUE TITLE
`export_bioimageio()` causes config file error

### ISSUE THREAD

**Issue by qin-yu** [stardist/stardist#252]
**Describe the bug**
The current `export_bioimageio()` function generates a config file containing an erroneous line of code, leading to errors. e.g. if I install a minimal conda environment for exporting StarDist model, the default environment.yaml file generated by `export_bioimageio()` contains a line `- bioimageio.core>=0.5.0; extra == "bioimageio"`, which causes a failed check in bioimage.io after submitting the exported model.

```pytb
error': 'Failed to install conda environment:\n'
            'channels: [defaults, conda-forge]\n'
            'dependencies:\n'
            '- python>=3.7,<3.8\n'
            '- bioimageio.core>=0.5.0; extra == "bioimageio"\n'
            '- pip\n'
            "- pip: [stardist>=0.8.3, 'tensorflow>=1.[15](https://github.com/bioimage-io/collection-bioimage-io/actions/runs/6476544517/job/17585511761?pr=648#step:9:16),<2']\n"
            "name: '8429[20](https://github.com/bioimage-io/collection-bioimage-io/actions/runs/6476544517/job/17585511761?pr=648#step:9:21)5'\n",
   'id': 'validation_summary_tensorflow_saved_model_bundle',
   'name': 'install test environment',
   'status': 'failed'
```

**To reproduce**
Standard use of `export_bioimageio()` in a minimal environment causes the issue.

**Expected behavior**
`bioimageio.core`'s requirement should be `- bioimageio.core>=0.5.0`

**Environment (please complete the following information):**
 - StarDist version 0.8.3
 - TensorFlow version 1.15.3
 - OS: Linux

**Comment by qin-yu** [comment]
This will be fixed together with the next `export_bioimageio()` issue

**Comment by uschmidt83** [comment]
Seems like a bug, I have no idea if this was always wrong or has changed at some point.

Sorry for the very late reply.

**Comment by uschmidt83** [comment]
Btw, this bug does not occur in one of my dev environments which uses an old version of `setuptools` (49.6.0.post20200814).

**Comment by uschmidt83** [comment]
Fixed by #254.
